src/redux/books/Books.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Books = () => (
+  <div>
+    <div className="book-list" />
+    <br />
+    <form>
+      <h2>ADD NEW BOOK</h2>
+      <input placeholder="Book title" />
+      <br />
+      <input placeholder="Category" />
+      <br />
+      <button type="submit">Add Book</button>
+    </form>
```

Comment:
- [ ] **[REQUIRED CHANGES]** You are not closing the `book-list div` tag. Can you fix it for us?

Reply:
Since the book-list div tag is empty, it is self-closing as I've written. To add the closing tag will bring linters errors.